Dashboard/app/js/lib/views/surveys/question-view.js

```diff
@@ -113,11 +113,15 @@ FLOW.QuestionView = FLOW.View.extend({
 	}
   }.property('this.type').cacheable(),
 
-  amNoOptionsType: function () {
+  hasExtraSettings: function(){
     var val;
     if (!Ember.none(this.type)) {
       val = this.type.get('value');
-      return val === 'PHOTO' || val === 'VIDEO' || val === 'DATE' || val === 'SIGNATURE';
+      if(val === 'PHOTO' || val === 'VIDEO' || val === 'DATE' || val === 'SIGNATURE'){
```

Comment:
the condition is `{{#if hasExtraSettings}}{{/if}}` and there are a number of if statements following for the question types that have extra settings. Its probably cleaner here to simply put that list e.g. 

`return this.get('amOptionType') || this.get('amFreetextType') || .... ;`

and that way if any type doesn't match the result will be false.